### PR TITLE
Pass storage target configuration to RDB Shredder

### PR DESCRIPTION
**Pull request by chuwy**

### CODE REVIEW

**Comment by chuwy** [comment]
Hey @dilyand, @BenFradet I'm planning to include it into R116 (EER 0.35.0). @benjben told me this is a Dilyan's collector release and I see it is okay if it will be the only addition to EER in this release. Do you guys have any objections?

File: 3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb [hunk 1]
```diff
@@ -1265,6 +1269,18 @@ def get_hdfs_expunge_step
       def get_processing_manifest(targets)
         targets[:ENRICHED_EVENTS].select { |t| not t.data[:processingManifest].nil? }.map { |t| t.data.dig(:processingManifest, :amazonDynamoDb, :tableName) }.first
       end
+
+      Contract ArrayOf[Iglu::SelfDescribingJson], String => Hash
+      def self.get_rdb_shredder_target(targets, shred)
```
Comment (benjben) [inline 3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb]: This function is pure, you could add a unit test.
You can find some examples [here](https://github.com/snowplow/snowplow/tree/master/3-enrich/emr-etl-runner/spec/snowplow-emr-etl-runner)

**Comment by dilyand** [comment]
Sure, go ahead @chuwy .

**Comment by chuwy** [comment]
This should have been merged in #4099

**Pull request closed without merging**